http-server/src/main/java/org/triplea/server/error/reporting/ErrorReportControllerFactory.java

```diff
@@ -31,7 +31,6 @@ public static ErrorReportController buildController(
             CreateIssueStrategy.builder()
                 .githubIssueClient(githubIssueClient)
                 .responseAdapter(new ErrorReportResponseConverter())
-                .isProduction(configuration.isProd())
```

Comment:
Is the `isProd()` method still being used?

Reply:
Yes, notably email sender still uses it to fake email delivery.